src/assets/custom/css/custom-softmod-feature.scss

```diff
@@ -0,0 +1,751 @@
+---
+# Front matter comment to ensure Jekyll properly reads file.
+---
+
+/* SoftMod Section
+   backgroundimages set in custom-layout-2.scss
+   ------------- */
+
+/********************************
+    Helpers - Text
+********************************/
+
+.text-900 {
+    font-weight: 900;
+}
+
+.text-700 {
+    font-weight: 700;
+}
+
+.text-navy {
+    color: #0E4072;
+}
+
+.text-white {
+    color: #FFFFFF;
+}
+
+.text-coral {
+    color: #FA7268;
+}
+
+.mx-width-600 {
+    max-width: 600px;
+}
+
+@media (min-width: 768px) {
+    .text-center--bp768 {
+        text-align: center;
+    }
+}
+
+/********************************
+    Helpers - Shadow
+********************************/
+
+.box-shadow {
+    box-shadow: 0 1px 3px rgba(0,0,0,0.12), 0 1px 2px rgba(0,0,0,0.24);
+}
+
+/********************************
+    Helpers - Spacing
+********************************/
+
+.pad-top-none {
+    padding-top: 0!important;
+}
+
+@media (min-width: 992px) {
+
+    .g-mt-neg100--desk {
+        top: -100px!important;
+        position: relative;
+    }
+
+    .g-pt-0--desk {
+        padding-top: 0px!important;
+    }
+
+    .g-pt-50--desk {
+        padding-top: 50px!important;
+    }
+
+    .g-pt-100--desk {
+        padding-top: 100px!important;
+    }
+
+    .g-pb-25--desk {
+        padding-bottom: 25px!important;
+    }
+
+    .g-pb-50--desk {
+        padding-bottom: 50px!important;
+    }
+
+    .g-pb-100--desk {
+        padding-bottom: 100px!important;
+    }
+
+    .g-pb-200--desk {
+        padding-bottom: 200px!important;
+    }
+
```

Comment:
Really not enjoying these abstract names `g-pb-200` what?

Reply:
These all come with the theme we are using. I've tried to use existing classes as much as possible and not introduce new ones and then follow the naming convention that is already set out. Not sure what g means but bp is padding bottom.